### PR TITLE
Add more UI elements

### PR DESCRIPTION
**Pull request by adityahemanth**

### CODE REVIEW

File: app/src/main/java/com/bhakti/ui/SummaryFragment.java [hunk 1]
```diff
@@ -27,29 +35,49 @@ public View onCreateView(LayoutInflater inflater,
     RecyclerView godsView = view.findViewById(R.id.recycler_view_gods);
     godsView.setHasFixedSize(true);
 
-    GodsModel ganesh = new GodsModel();
-    ganesh.songCount = 23;
-    ganesh.title = "Ganesh";
+    SongsTable songsTable = new SongsTable(getContext());
+    SongInfoRequest.Builder builder = SongInfoRequest.builder();
```
Comment (sasi306) [inline app/src/main/java/com/bhakti/ui/SummaryFragment.java]: nit: SongInfoRequestBuilder.
I always find builder hard to relate to.

File: app/src/main/java/com/bhakti/ui/SummaryFragment.java [hunk 2]
```diff
@@ -27,29 +35,49 @@ public View onCreateView(LayoutInflater inflater,
...
     LayoutManager layoutManager = new LinearLayoutManager(getContext(),
         LinearLayoutManager.HORIZONTAL, false);
     godsView.setLayoutManager(layoutManager);
     godsView.setAdapter(godsViewAdapter);
 
     return view;
   }
+
+  private List<GodsModel> buildGodsModels(List<SongInfo> songInfos) {
+    List<Gods> indexList = new ArrayList<>();
```
Comment (sasi306) [inline app/src/main/java/com/bhakti/ui/SummaryFragment.java]: nit: godIndexList

File: app/src/main/java/com/bhakti/ui/SummaryFragment.java [hunk 3]
```diff
@@ -9,47 +9,120 @@
...
-    GodsModel shiva = new GodsModel();
-    shiva.title = "Shiva";
-    shiva.songCount = 12;
+    return view;
+  }
 
-    GodsModel lakshmi = new GodsModel();
-    lakshmi.title = "Lakshmi";
-    lakshmi.songCount = 19;
+  private void setupViews(View view) {
```
Comment (sasi306) [inline app/src/main/java/com/bhakti/ui/SummaryFragment.java]: Seperate setupViews functions for each section ?

File: app/src/main/java/com/bhakti/ui/SummaryFragment.java (hunk 1 above)
Comment (snmlabs) [reply app/src/main/java/com/bhakti/ui/SummaryFragment.java]: Done.

File: app/src/main/java/com/bhakti/ui/SummaryFragment.java (hunk 3 above)
Comment (snmlabs) [reply app/src/main/java/com/bhakti/ui/SummaryFragment.java]: Done.

**Pull request merged**